JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/agent/JPSAgent.java

```diff
@@ -54,5 +55,34 @@ public boolean validateInput(JSONObject requestParams) throws BadRequestExceptio
         }
         return true;
     }
-
+    
+    /**
+     * Execute a <a href="https://www.w3.org/TR/sparql11-query/">SPARQL Query</a> on the target resource 
```

Comment:
You can use 

`{@link <a href="https://www.w3.org/TR/sparql11-query/">}` 

in code comments for those things.

Reply:
Changed